jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/KubernetesHelperTest.java

```diff
@@ -53,23 +54,35 @@
 import io.fabric8.kubernetes.client.dsl.Resource;
 import io.fabric8.openshift.api.model.DeploymentConfigBuilder;
 import io.fabric8.openshift.api.model.Template;
-import mockit.Expectations;
-import mockit.Mocked;
-import mockit.Verifications;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
 
+@RunWith(MockitoJUnitRunner.class)
 public class KubernetesHelperTest {
 
     private KitLogger logger;
 
+    @Mock
+    private KubernetesClient kubernetesClient;
+
+    @Mock
+    private Resource<Service> svcResource;
```

Comment:
Neither the @RunWith(MockitoJUnitRunner.class) annotation
nor these fields are necessary since these variables are used locally and initialized in the test method

Reply:
@manusa I suppose we cant give parameters inside a test method , so I have removed those variables and declared above globally. If we remove@RunWith(MockitoJUnitRunner.class) in this class test cases are failing.

Reply:
Please verify KubernetesHelperTest now! @manusa